Anytype/Sources/Analytics/AnytypeAnalytics/AnytypeAnalyticsCore.swift

```diff
@@ -20,7 +20,7 @@ actor AnytypeAnalyticsCore {
     private var userProperties: [AnyHashable: Any] = [:]
     
     @Injected(\.participantSpacesStorage)
-    private var participantSpacesStorage: ParticipantSpacesStorageProtocol
+    private var participantSpacesStorage:any ParticipantSpacesStorageProtocol
```

Comment:
`any` is part of type. Maybe add space between `:` and `any`?

Reply:
like in this pr https://github.com/anyproto/anytype-swift/pull/1705/files#diff-e43ba33fa211b5143e294d85199aeb4fec99e9dfd4f40a392c370061a4329f6cR36

Reply:
Yep, it was just messy replace action, fixed